public/app/features/explore/RichHistory/RichHistory.test.tsx

```diff
@@ -2,14 +2,25 @@ import { render, screen } from '@testing-library/react';
 import React from 'react';
 
 import { GrafanaTheme } from '@grafana/data';
+import { SortOrder } from 'app/core/utils/richHistory';
 
-import { SortOrder } from '../../../core/utils/richHistoryTypes';
 import { ExploreId } from '../../../types/explore';
 
 import { RichHistory, RichHistoryProps, Tabs } from './RichHistory';
 
 jest.mock('../state/selectors', () => ({ getExploreDatasources: jest.fn() }));
 
+jest.mock('@grafana/runtime', () => ({
+  ...(jest.requireActual('@grafana/runtime') as unknown as object),
```

Comment:
```suggestion
  ...jest.requireActual('@grafana/runtime'),
```

i don't think we need `as unknown as object` here, do we?

Reply:
Oh totally! I copy-pasted without thinking 🙈 Updated in 01d3c34eee6438605d49e11f2337147bf4887ae2